src/mono_font/mono_text_style.rs

```diff
@@ -386,7 +386,7 @@ pub struct MonoTextStyleBuilder<'a, C> {
 
 impl<C> MonoTextStyleBuilder<'_, C> {
     /// Creates a new text style builder.
-    pub fn new() -> Self {
+    pub const fn new() -> Self {
```

Comment:
The builder can't be used in a const context, because none of the other methods are `const`. I would leave the constructor as is until it is possible to make the entire builder work in a const context.

Reply:
Ok, reverted.